### PR TITLE
Check-in migration from wiki

### PR DESCRIPTION
**Pull request by enolfc**
# Summary 

----

**Related issue :**

### CODE REVIEW

**Comment by enolfc** [comment]
This moves most of the content from the wiki regarding AAI. It's missing:
https://wiki.egi.eu/wiki/AAI_expressing_vo_group_membership_and_role_information
and
https://wiki.egi.eu/wiki/AAI_FAQ

It also links this page as a success story: https://wiki.egi.eu/wiki/EGI_AAI_integration_with_ELIXIR_AAI which I'm not really sure it's still relevant/up to date. If so, it should be migrated also

File: content/en/providers/check-in/idp/_index.md [hunk 1]
```diff
@@ -0,0 +1,286 @@
+---
+title: "Identity Providers"
+description: "Check-in guide for Identity Providers"
+weight: 10
+type: "docs"
+---
+
+
```
Comment (gwarf) [inline content/en/providers/check-in/idp/_index.md:8]:
```suggestion
```

File: content/en/providers/check-in/idp/_index.md [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+This page contains information about integrating your identity
+provider with [Check-in](https://www.egi.eu/services/check-in/)
+in order to allow users in your community to access EGI tools and
+services.
+
+Organisations who want to register their IdP in Check-in needs to fill
+this [form](https://documents.egi.eu/document/3086) in case the IdP is
+not publishing R&S and Sirtfi compliance in eduGAIN. A PDF scan of a
+printed and signed copy should be sent to `operations_at_egi.eu`
+
```
Comment (gwarf) [inline content/en/providers/check-in/idp/_index.md:18]:
```suggestion
```

File: content/en/providers/check-in/idp/_index.md [hunk 3]
```diff
@@ -0,0 +1,286 @@
+---
+title: "Identity Providers"
+description: "Check-in guide for Identity Providers"
+weight: 10
+type: "docs"
+---
+
+
+This page contains information about integrating your identity
+provider with [Check-in](https://www.egi.eu/services/check-in/)
```
Comment (gwarf) [inline content/en/providers/check-in/idp/_index.md:10]:
```suggestion
provider (IdP) with [Check-in](https://www.egi.eu/services/check-in/)
```

File: content/en/providers/check-in/idp/_index.md [hunk 4]
```diff
@@ -0,0 +1,286 @@
...
+in order to allow users in your community to access EGI tools and
+services.
+
+Organisations who want to register their IdP in Check-in needs to fill
+this [form](https://documents.egi.eu/document/3086) in case the IdP is
+not publishing R&S and Sirtfi compliance in eduGAIN. A PDF scan of a
```
Comment (gwarf) [inline content/en/providers/check-in/idp/_index.md:16]:
```suggestion
not publishing REFEDS [R&S](https://refeds.org/category/research-and-scholarship) and [Sirtfi](https://refeds.org/sirtfi compliance in eduGAIN. A PDF scan of a
```

**Review by gwarf — APPROVED** [review]
Thanks Enol!
I started to suggest a few fixes, but it may be better to plan a real review later, so I would propose to address the suggestions I made then to push it.

File: content/en/providers/check-in/idp/_index.md [hunk 5]
```diff
@@ -0,0 +1,284 @@
...
+---
+
+This page contains information about integrating your identity
+provider (IdP) with [Check-in](https://www.egi.eu/services/check-in/)
+in order to allow users in your community to access EGI tools and
+services.
+
+Organisations who want to register their IdP in Check-in needs to fill
+this [form](https://documents.egi.eu/document/3086) in case the IdP is
+not publishing REFEDS [R&S](https://refeds.org/category/research-and-scholarship) and [Sirtfi](https://refeds.org/sirtfi compliance in eduGAIN. A PDF scan of a
```
Comment (enolfc) [inline content/en/providers/check-in/idp/_index.md:15]:
```suggestion
not publishing REFEDS [R&S](https://refeds.org/category/research-and-scholarship)
and [Sirtfi](https://refeds.org/sirtfi) compliance in eduGAIN. A PDF scan of a
```

**Review by enolfc — COMMENTED** [review]
Self request for changes

**Comment by enolfc** [comment]
> I started to suggest a few fixes, but it may be better to plan a real review later, so I would propose to address the suggestions I made then to push it.

Fine with me. Suggestions applied already

**Pull request merged**